mod-audit-server/src/main/java/org/folio/rest/impl/AuditDataAcquisitionImpl.java

```diff
@@ -78,11 +83,30 @@ public void getAuditDataAcquisitionOrderLineById(String orderLineId, String sort
     });
   }
 
+  @Override
+  public void getAuditDataAcquisitionPieceById(String pieceId, String sortBy, AuditDataAcquisitionPieceIdGetSortOrder sortOrder,
+                                               int limit, int offset, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    LOGGER.debug("getAuditDataAcquisitionOrderById:: Retrieving Audit Data Acquisition Piece By Id : {}", pieceId);
+    String tenantId = TenantTool.tenantId(okapiHeaders);
+
+    vertxContext.runOnContext(c -> {
```

Comment:
It is just a standard controller, I think runOnContext is not necessary here, we do not wrap in this method our standard acquisition controllers, I think we can delete from this controller as well (for all 3 methods)

Reply:
working...

Reply:
removed